models/us_census_cps/code/upload.py

```
@@ -0,0 +1,105 @@
+def upload_table(slug: str, expected_rows: int | None) -> int:
+    path = PARQUET_ROOT / slug
+    if not path.exists():
+        raise FileNotFoundError(f"Missing parquet path: {path}")
+
+    tb = bd.Table(dataset_id=DATASET_ID, table_id=slug)
+
+    # Delete stale GCS staging prefix (avoids BQ partition key conflicts)
+    st = bd.Storage(dataset_id=DATASET_ID, table_id=slug)
+    try:
+        st.delete_table(mode="staging", not_found_ok=True)
+    except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)